common/utilities/tests/test_logger.py

```diff
@@ -162,3 +163,13 @@ def test_write_throws_error_on_bad_params(self):
     def test_undefined_log_ref_throws_error(self):
         with self.assertRaises(ValueError):
             log.IntegrationAdaptorsLogger('')
+
+    @patch('sys.stdout', new_callable=io.StringIO)
+    def test_correct_time_format(self, mock_stdout):
```

Comment:
Don't forget to raise some techdebt for figuring out the issue where this test name actually affects the test passing/failing.